docs/How-To-Run-Query-on-IDP-DB-In-Ruby-Console.rst

```diff
@@ -0,0 +1,98 @@
+.. _how-to-run-query-on-idp-db-in-ruby-console:
+
+How To Run a Custom Query on IDP DB in Ruby Console
+==============================================================================
+
+.. contents::
+    :local:
+
+
+1. Get the Utility Shell Scripts from ``devops`` Repo
+------------------------------------------------------------------------------
+
+- Clone the repo: https://github.com/18F/identity-devops
+- Setup your ruby following this guide: https://github.com/18F/identity-idp. Finally you should able to see your ruby version is 2.3.X by running ``ruby -v`` command.
+- Setup your PIV card auth, follow this guide: https://github.com/18F/identity-private/wiki/Operations:-MacOSX-PIV-to-SSH-key-extraction. This should be already be done as part of your on-boarding process.
+
+
+2. SSH to prod instance, log in to Ruby Console
+------------------------------------------------------------------------------
+
+1. Establish SSH to current IDP prod server (ec2 instance):
+
+first:
+
+.. code-block:: bash
+
+    cd <path-to-identity-devops>
+
+stay in root dir, and:
+
+.. code-block:: bash
+
+    ./bin/ssh-instance --newest asg-prod-idp
+
+2. Once you have access to the IDP server, then get in ruby console:
+
+first:
+
+.. code-block:: bash
+
+    cd /srv/idp/current
+
+then:
+
+.. code-block:: bash
+
+    sudo -uwebsrv RAILS_ENV=production ALLOW_CONSOLE_DB_WRITE_ACCESS=true bundle exec rails c
```

Comment:
suggest changing  ALLOW_CONSOLE_DB_WRITE_ACCESS=true to ALLOW_CONSOLE_DB_WRITE_ACCESS=false for read-only access by default as you do not want users to accidentally write anything to prod db.

Reply:
The only value it recognizes is 'true', otherwise it defaults to false. Ideally the `ALLOW_CONSOLE_DB_WRITE_ACCESS` flag would be left off.